src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1569,4 +1571,96 @@ private function addNotifierSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addSchedulerSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('scheduler')
+                    ->info('Scheduler configuration')
+                    ->{!class_exists(FullStack::class) && class_exists(SchedulerInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                    ->children()
+                        ->scalarNode('export_directory')
+                            ->info('The path where the tasks should be exported')
+                            ->defaultValue('%kernel.project_dir%/var/scheduler/exports')
+                        ->end()
+                        ->scalarNode('output_path')
+                            ->info('The path where the output should be sent')
+                            ->defaultValue('%kernel.project_dir%/var/scheduler')
+                        ->end()
+                        ->scalarNode('path')
+                            ->info('The path used to trigger tasks using http request')
+                            ->defaultValue('/_tasks')
+                        ->end()
+                        ->scalarNode('timezone')
+                            ->defaultValue('Europe/Paris')
```

Comment:
Maybe `utc` would fit better?

Reply:
Thanks for the spot, fixed on the latest commit 🙂